client/reader/stream/reader-tag-sidebar/index.jsx

```diff
@@ -0,0 +1,46 @@
+import { useTranslate } from 'i18n-calypso';
+import TagLink from 'calypso/blocks/reader-post-card/tag-link';
+import { useRelatedMetaByTag } from 'calypso/data/reader/use-related-meta-by-tag';
+import ReaderListFollowingItem from 'calypso/reader/stream/reader-list-followed-sites/item';
+import '../style.scss';
+
+const ReaderTagSidebar = ( { tag } ) => {
+	const translate = useTranslate();
+	const relatedMetaByTag = useRelatedMetaByTag( tag );
+	if ( relatedMetaByTag === undefined ) {
+		return null;
+	}
+
+	const tagLinks = relatedMetaByTag.data?.related_tags
+		.sort( ( a, b ) => b.score - a.score )
+		.map( ( relatedTag ) => <TagLink tag={ relatedTag } key={ relatedTag.slug } /> );
+
+	const relatedSitesLinks =
+		relatedMetaByTag.data?.related_sites && relatedMetaByTag.data.related_sites.length > 0
```

Comment:
nit: you could use only one line with the null check `?` character `relatedMetaByTag.data?.related_sites?.length > 0`

Reply:
Actually I think if you just did `relatedMetaByTag.data?.related_sites?.map()` you wouldn't even need the ternary statement at all, because if the length where 0, it would return nothing.